Client/WebFilters/WebsiteRedirects.swift

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+import BraveShared
+
+struct WebsiteRedirects {
+  private enum Site: CaseIterable {
```

Comment:
For architecture: considered making a struct here with few `rules` fields instead.
There's a chance users would like to add their own redirection rules in the future.

Reply:
I started with struct initially then switched to enum, changed it to struct again